mac/sound.cpp

```diff
@@ -103,24 +103,24 @@ void CSound::GetAvailableInOutDevices()
 
     // always add system default devices for input and output as first entry
     lNumDevs                 = 0;
-    strDriverNames[lNumDevs] = "System Default In/Out Devices";
+    strDriverNames[lNumDevs] = "System Default In/Out Devices"; // TODO: why can't this be translated?
 
     iPropertySize               = sizeof ( AudioDeviceID );
     stPropertyAddress.mSelector = kAudioHardwarePropertyDefaultInputDevice;
 
     if ( AudioObjectGetPropertyData ( kAudioObjectSystemObject, &stPropertyAddress, 0, NULL, &iPropertySize, &audioInputDevice[lNumDevs] ) )
     {
-        throw CGenErr ( tr ( "CoreAudio input AudioHardwareGetProperty call failed. "
-                             "It seems that no sound card is available in the system." ) );
+        throw CGenErr ( tr ( "No sound card is available in your system. "
```

Comment:
```suggestion
        throw CGenErr ( tr ( "No sound hardware is available in your system. "
```

Reply:
To be discussed. Only open question on this PR. Therefore waiting for a concensus.

Reply:
Shouldn't really use passive voice if we can help it. "Your system has no available sound hardware" would be the preferred grammar.